web/client/components/app/__tests__/StandardRouter-test.jsx

```diff
@@ -209,6 +209,6 @@ describe('StandardRouter', () => {
         setTimeout(() => {
             expect(spyLoad).toHaveBeenCalled();
             done();
-        }, 0);
+        }, 200);
```

Comment:
Only a suggestion: If this is the only check we do in this test, maybe calling `done` in the callback instead of using spy can make the thing less dangerous?

Reply:
uhmm... could work, let me check